File: 3.js
let amount = parseInt(prompt()); // ввод количества чисел
for (let i = 0; i < amount; i++) {
    let number = parseInt(prompt()); // ввод чисел
    let count = 0;
    while (number != 0) {
        if ((number % 10) % 2 == 0) { // проверка каждой цифры на чётность
            count++; // подсчёт количества
        }
        number = Math.floor(number / 10);
    }
    process.stdout.write("Result: " + count + "\n");
}

File: 3.kt
import java.util.Scanner;

fun main() {
    val scanner = Scanner(System.`in`)
    val amount = scanner.nextInt() // ввод количества чисел
    for (i in 1..amount) {
        var number = scanner.nextInt() // ввод чисел
        var count = 0
        while (number != 0) {
            if ((number % 10) % 2 == 0) // проверка на чётность цифр
                count++ // подсчёт количества
            number /= 10
        }
        println("Result: " + count)
    }
}